backend/Match/managers.py

```diff
@@ -0,0 +1,73 @@
+from django.db import models, transaction, IntegrityError
+from django.db.models import Q
+
+from .PlayerMatchStatus import PlayerMatchStatus
+from Player.models import Player
+from User.models import User
+
+import sys
+
+class MatchManager(models.Manager):
+	def create_match_and_its_players(self, left_side_user_id, right_side_user_id):
```

Comment:
As discussed in the meeting, I think the logic of creating the players and the match should be moved to some utility / shared manager where then we can import both User / Player and the Match. These two entities kind of depend on each other and we should decouple them to be able scale the business logic later. So, instead of the Player being dependent on a match or match being dependent on a player, they rather should both be dependent on some PlayerMatchManager!

Reply:
sure, sounds good